cypress/integration/frontend-testing.js

```diff
@@ -0,0 +1,15 @@
+function runFrontendTest(name) {
+    it(name, () => {
+        cy.window().then((win) => {
+            return win.frontendTesting.run(name);
+        });
+    });
+}
+
+describe('Frontendtestresults', () => {
+    before(() => {
+        cy.visit('/');
+    });
+
+    runFrontendTest('HelloWorld');
```

Comment:
I like the general idea here, but maybe we could just grab all the tests from window.frontendTesting? We're basically just deduplicating the test name here (and in the test class).

Reply:
ah right, that makes sense

Reply:
I just tried to get this to work, but I'm not sure it's possible because of how cypress works.
(you can only run `cy.window()` from an `it()` and `cy.window()` is async, and you can't run other tests inside another `it()`, and various tries with passing on promises did not work either, etc)

Reply:
I see, this should be good then!